src/www/vpn_ipsec_mobile.php

```diff
@@ -524,6 +524,7 @@ function print_legacy_box($msg, $name, $value)
                             28 => '28 (Brainpool EC 256 bits)',
                             29 => '29 (Brainpool EC 384 bits)',
                             30 => '30 (Brainpool EC 512 bits)',
+                            31 => '31 (Elliptic Curve 25519)',
```

Comment:
```suggestion
                            31 => '31 (Curve25519)',
```

See https://cr.yp.to/ecdh.html - the page of the developer of this curve -> it is spelled this way.

Reply:
please use the https://wiki.strongswan.org/projects/strongswan/wiki/IKEv2CipherSuites doc to refer to, otherwise it's confusing

Reply:
@fichtner ok, then keep it as is since the official documentation of strongswan differs from the official documentation of the cipher.